packages/utils/src/npm.ts

```diff
@@ -113,17 +127,16 @@ const generateSearchQuery = (searchTerm: string, from?: number, size = 100) => {
  * Searches all the packages in npm given `searchTerm`.
  */
 export const search = async (searchTerm: string): Promise<NpmPackage[]> => {
-    const result = await npmRegistryFetch.json(generateSearchQuery(searchTerm)) as NpmSearchResults;
+    const result = (await npmRegistryFetch.json(generateSearchQuery(searchTerm))) as NpmSearchResults;
```

Comment:
Do we need now the extra parems?

Reply:
Added for readability (by Prettier). Should I remove those if they do not conform to style guide?

Reply:
My vote is to leave them - I think having the extra parens here makes the intent clearer.

Reply:
OK